docs/api/commands/origin.mdx

```diff
@@ -572,5 +550,6 @@ inclusion in a future version of Cypress.
 
 - [Easily test multi-domain workflows with cy.origin](https://cypress.io/blog/2022/04/25/cypress-9-6-0-easily-test-multi-domain-workflows-with-cy-origin/)
 - [Custom Commands](/api/cypress-api/custom-commands)
+- [`Cypress.require()`](/api/cypress-api/require)
```

Comment:
Can we add a history note about updates to use Cypress.require()?

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/5020/commits/257a803e312e274f0e654dbe73ac7cc883a88d21